content/applications/inventory_and_mrp/purchase/manage_deals/control_bills.rst

```diff
@@ -1,60 +1,117 @@
-generate the draft bill. If you try to create one without having received any product, you get an
-error message.
+In the Purchase app, create a new Request for Quotation. Fill out the information on the quotation,
+add products to the invoice lines, and click :guilabel:`Confirm Order`. Then, click on the
+:guilabel:`Receipt smart button`. Set the quantities in the *Done* column to match the quantities
+in the *Demand* column, and :guilabel:`Validate`. Then, in the purchase order, click
+:guilabel:`Create Bill` and :guilabel:`Confirm`. Since the policy is set to *Received quantities*,
+the *Draft Bill* can be confirmed *only* when at least some of the quantities are received.
 
-.. image:: control_bills/control-bills-no-invoiceable-line.png
-   :align: center
-   :alt: No invoiceable line error message in Odoo Purchase
+3-way matching
+==============
 
-Change a specific product's control policy
-------------------------------------------
+Activating 3-way matching ensures that vendor bills are only paid once some or all of the products
+included in the purchase order have actually been received. To activate it, go to
+:menuselection:`Purchase --> Configuration --> Settings`, and scroll down to the **Invoicing**
+section. Then, click :guilabel:`3-way matching: purchases, receptions, and bills`.
 
-If you want to modify a specific product's control policy, go to :menuselection:`Products -->
-Products`, open it, click on *Edit* and go to the *Purchase tab*. There you can change a product's
-default bill control policy.
+.. note::
+   3-way matching is intended to work with the bill control policy set to *Received quantities*.
+
+Know when to pay vendor bills with 3-way matching
+-------------------------------------------------
 
-.. image:: control_bills/control-bills-quantities-on-product.png
+When 3-way matching is activated, vendor bills will display the :guilabel:`Should Be Paid` field
+under the :guilabel:`Other Info` tab. When a new vendor bill is created, the field will be set to
+*Yes*, since a bill can't be created until at least some of the products included in a purchase
+order have been received.
+
+.. image:: control_bills/vendor-bill-should-be-paid.png
    :align: center
-   :alt: Vendor bills default control setting for new products in Odoo Purchase
+   :alt: Draft bill should be paid field status.
+
+.. note::
+   If the total quantity of products from a purchase order has not been received, Odoo only
+   includes the products that *have* been received in the draft vendor bill.
+
+*Draft Bills* can be edited to increase the billed quantity, change the price of the products in
+the bill, and add additional products to the bill. If this is done, the :guilabel:`Should Be Paid`
+field status will be set to *Exception*. This means that Odoo notices the discrepancy, but doesn't
+block the changes or display an error message, since there might be a valid reason for making
+changes to the draft bill.
```

Comment:
```suggestion
Draft bills can be edited to increase the billed quantity, change the price of the products in
the bill, and add additional products to the bill. If this is done, the :guilabel:`Should Be Paid`
field status will be set to :guilabel:`Exception`. This means that Odoo notices the discrepancy, but
doesn't block the changes or display an error message, since there might be a valid reason for
making changes to the draft bill.
```